contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,82 @@
+# Validation Suite Testing in Kubernetes
```

Comment:
Overall comment, and extending what @timothysc mentioned below:
I find the general taxonomy of e2e tests to be confusing.  Even if we ignore "Profiles" for the moment, the relationship between:

1. e2e tests (generally)
2. Validation test suites
3. Conformance tests
4. All the other categories and sub-categories of e2e tests that we have

is not at all clear to me based on the descriptions given in this doc.  This is further compounded by the somewhat ambiguous naming -- the dictionary distinction between 'Validation' ("the action of making or declaring something legally or officially acceptable") and 'Conformance' ("compliance with standards, rules, or laws") is very blurry indeed.

I added more comments around this below.

Reply:
I'm adding more clarity to distinguish between validation and conformance, incorporating many of the explanations provided by @dims

Reply:
FYI, I'm not sure I agree with @Dims on that 😁

Reply:
LOL sometimes i don't agree with myself :)

Reply:
Catching up. It looks like this still wasn't resolved?

We need to tests interfaces of extension points and of individual pluggable components. Many of those won't be end-to-end tests. That's what I thought we would call validation suites.

Conformance tests are a subset of end-to-end tests.

Reply:
@bgrant0607  Sorry I am trying to understand this. If you do not want to call them as 'subset of end-to-end' tests, my assumption was that we could trigger a validation suite by just using a proper regex in the ginkgo.focus. We can refine the statement saying 'Validation Suite tests are similar e2e tests and may be part of or extension to existing e2e tests'?